spec/mailers/notifier_spec.rb

```diff
@@ -131,7 +157,31 @@
     it 'can handle a reshare' do
       reshare = FactoryGirl.create(:reshare)
       like = reshare.likes.create!(:author => bob.person)
-      mail = Notifier.liked(alice.id, like.author.id, like.id)
+      Notifier.liked(alice.id, like.author.id, like.id)
+    end
+  end
+
+  describe ".liked limited" do
+    before do
+      @post = FactoryGirl.create(:status_message, :author => alice.person, :public => false)
+      @like = @post.likes.create!(:author => bob.person)
+      @mail = Notifier.liked(alice.id, @like.author.id, @like.id)
+    end
+
+    it 'TO: goes to the right person' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.